inspire/modules/workflows/tasks/submission.py

```diff
@@ -121,7 +124,7 @@ def _reply_ticket(obj, eng):
         from flask import render_template
 
         ticket_id = obj.extra_data.get("ticket_id", "")
-        if not ticket_id:
+        if not ticket_id or not cfg.get("PRODUCTION_MODE"):
```

Comment:
On second thought, I am not sure we need the production mode check in `reply_ticket` as `ticket_id` would in principle be empty as `create_ticket` will not set `ticket_id` if not in production mode.